game/world/managers/objects/units/creature/CreatureManager.py

```diff
@@ -533,8 +533,16 @@ def _perform_random_movement(self, now):
 
     def _perform_combat_movement(self):
         if self.combat_target:
+
+            # Fixes issue #234
+            # If target is dead > evade
+            if self.combat_target.is_alive == False:
```

Comment:
There might be more targets attacking this unit, check attack_update() first lines.

Reply:
Oh yes, here I was too eagerly pursuing my first ever push to open source.. Gonna check that out.

Reply:
You can still fix #234 👍 , just forget the timer thing.